vue-frontend/src/components/jobs/VirtuesView.vue

```diff
@@ -0,0 +1,397 @@
+<template>
```

Comment:
Can't the component be just Virtues?

Reply:
Vue does not support single word for components due to prevent conflicts with standard HTML tags